### ISSUE TITLE
Broken profile page

### ISSUE THREAD

**Issue by timmersuk** [KSP-SpaceDock/SpaceDock#135]
http://spacedock.info/profile/timmers_uk is returning an internal server error when I access it

This almost certainly happened as a result of deleting one of my mods from SpaceDock as it looked (on the profile page) as if I had added it twice - once as a 1.0.5 compat mod, once as 1.1.

Once I deleted the mod, my profile page then started erroring, and still does. I recreated the KeepFit mod which is the mod in question, as it looks as if it wasn't a dupe after all, but I suspect there could still be internal shenanigans going on as a result of the delete.

Hope this is of some help - simple enough hopefully to access my profile page and replicate the issue.

**Comment by V1TA5** [comment]
computer sys no:
FileNotFoundError: [Errno 2] No such file or directory: /timmers_uk_525/KeepFit/KeepFit-1459360883.3183715.png'
i could put some dummy img there as quick fix :)

**Comment by V1TA5** [comment]
i just copied the file you allready had on there and its working again :)

**Comment by timmersuk** [comment]
I broke it again, then fixed it again - I had duped the KeepFit mod by mistake. When I deleted the new one (again), my profile was back to dishing out 500 errors, as did doing a search for KeepFit.

I reploaded the banner image and that fixed it.

Looking at the previous comment, I suspect maybe there wasn't uniqueness in the serverside filename for the uploaded image between the 2 mods, because of same user, same mod name? Possibly some defensive coding to ensure the banner image files are unique on the mod id rather than or in addition to the name? (or I should stop failing?)

tnx.
